client/galaxy/scripts/components/Upload/Composite.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+    <upload-wrapper wrapperClass="upload-view-composite">
+        <div class="upload-helper" v-show="showHelper">Select extension to show files required to upload</div>
```

Comment:
Maybe "Select Composite Type to show required files." instead since the select field is labeled: "Composite Type".

Reply:
Good catch - made the change.